src/vm/safehandle.cpp

```diff
@@ -50,6 +50,11 @@ void SafeHandle::Init()
     s_ReleaseHandleMethodSlot = pMD->GetSlot();
 }
 
+// These AddRef and Release methods (and supporting functions) also exist with equivalent
+// code in SafeHandle.cs.  Those implementations are the primary ones used by most code
+// and exposed publicly; the implementations here are only for use by the runtime, without
+// having to call out to the managed implementations.
```

Comment:
This seemed like a reasonable compromise.  Let me know if you disagree.

Reply:
👍